civihr_employee_portal/views/views_export/views_hr_resource_types_list.inc

```diff
@@ -26,6 +26,14 @@ $handler->display->display_options['exposed_form']['type'] = 'basic';
 $handler->display->display_options['pager']['type'] = 'none';
 $handler->display->display_options['pager']['options']['offset'] = '0';
 $handler->display->display_options['style_plugin'] = 'table';
+/* Header: Global: Text area */
+$handler->display->display_options['header']['area']['id'] = 'area';
+$handler->display->display_options['header']['area']['table'] = 'views';
+$handler->display->display_options['header']['area']['field'] = 'area';
+$handler->display->display_options['header']['area']['label'] = 'Add HR Resource Type';
+$handler->display->display_options['header']['area']['empty'] = TRUE;
+$handler->display->display_options['header']['area']['content'] = '<a class="btn btn-default btn-primary" href="/admin/structure/taxonomy/hr_resource_type/add"><span class="fa fa-plus"></span> Add HR Resource Type</a>';
```

Comment:
I think we should add this button using a template instead. It would give us more flexibility to change its position/styling without having to update the view itself

Reply:
do you have any example for a view that use template since I am not really that familiar with such stuff?

Reply:
Here is an example of how it is used by the staff directory view: https://github.com/compucorp/civihr-employee-portal/blob/c6ac2e215f87c868b7345bd9e6755d4774e960ab/civihr_employee_portal/views/templates/views-view--civihr-staff-directory--page.tpl.php

Better confirm this with a Drupal dev, but if I'm not wrong, the file follows this structure:
`views-view--<view name>--<display name>.tpl.php`